tests/test_pymatgen_molecule.py

```diff
@@ -0,0 +1,20 @@
+import os
+import numpy as np
+import unittest
+from context import dpdata
+
+class TestPOSCARCart(unittest.TestCase):
+    
+    def setUp(self): 
+        self.system = dpdata.System()
+        self.system.from_pymatgen_molecule(os.path.join('pymatgen', 'FA-001.xyz'))
+
+    def test_to_molecule(self):
+        mols = self.system.to_pymatgen_molecule()
+        print(mols[-1])
```

Comment:
[`assertEqual`](https://docs.python.org/3/library/unittest.html#assert-methods) is preferred to just printing it. For example, to assert it not empty, you can use:
```suggestion
        self.assertEqual(len(mols), 1)
```